docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+services:
+  integration_tests:
+    build:
+      dockerfile: Dockerfile
+      context: ${GSY_E_TARGET_REPO:-./}
+    container_name: ${CONTAINER_NAME:-gsy-e-tests}
+    entrypoint: ["gsy-e"]
+    command: ${COMMAND:-}
+    environment:
+      - REDIS_URL=${REDIS_URL:-redis://redis.container:6379/}
+    volumes:
+      - ${GSY_E_TARGET_REPO:-integration_tests}:/gsy-e
+
+  redis:
+    image: redis:6.2.5
+    restart: always
+    container_name: gsy-e-tests-redis
+    ports:
+      - "${PORT:-3679}:${PORT:-6379}"
+    hostname: ${HOST:-redis.container}
```

Comment:
I'm afraid that `PORT` and `HOST` names might be too broad and they could clash with some other env variable.

Reply:
> I'm afraid that PORT and HOST names might be too broad and they could clash with some other env variable. Or am I wrong?

I agree with your concern. This would be fixed with more variable verbosity. However, as you then mentioned, these will hardly change, and we might want to set them explicitly in sake of simplicity.
I decided to add the flexibility just because the steps which build the image seemed to have that flexibility as well.

I would also go for setting those explicitly.

Reply:
What's the progress on this? I think that the options are two AFAIU: either we rename these using more verbose names, or we just hardcode their values here without using variables.

Reply:
I adapted to this comments on the compose file on GBIT's PR:
https://github.com/gridsingularity/gsy-backend-integration-tests/pull/33 

I opted for hardcoding the values.